docs/installation/gce-kubernetes.md

```diff
@@ -110,22 +110,20 @@ _You can delete the instance if your not planning to use it for anything else. B
 	The response would be similar to
 
 	```
-	address: 123.123.123.123
-	creationTimestamp: '2017-05-16T05:26:24.894-07:00'
-	description: ''
-	id: '1234556789'
-	kind: compute#address
-	name: test
-	selfLink: https://www.googleapis.com/compute/v1/projects/eventyay/global/addresses/test
-	status: RESERVED
+	Created [https://www.googleapis.com/compute/v1/projects/eventyay-212514/regions/us-west1/addresses/testip].
 	```
+- Listing static external IP addresses
 
-	Note down the address. (In this case `123.123.123.123`). We'll call this **External IP Address One**.
+    ```bash
+	gcloud compute addresses list
```

Comment:
This will list all IP addresses assigned to other google compute engines too. We only want the IP address for the cluster that was created

Reply:
@niranjan94 PR updated. Please review